python/test/block/Test_ACEtk_KalbachMannDistributionData.py

```diff
@@ -0,0 +1,131 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from ACEtk import KalbachMannDistributionData
+from ACEtk import TabulatedKalbachMannDistribution
+
+class Test_ACEtk_KalbachMannDistributionData( unittest.TestCase ) :
+    """Unit test for the KalbachMannDistributionData class."""
+
+    chunk = [            0,             2,  1.219437E+01,  2.000000E+01,
+                        27,            39,             1,             2,
+              0.000000E+00,  1.866919E-02,  5.356419E+01,  0.000000E+00,
+              0.000000E+00,  1.000000E+00,  0.000000E+00,  0.000000E+00,
+              2.391154E-01,  2.398743E-01,             2,             3,
+              0.000000E+00,  1.120151E+00,  7.592137E+00,  7.738696E-02,
+              4.209016E-01,  1.226090E-11,  0.000000E+00,  5.382391E-01,
+              1.000000E+00,  2.491475E-03,  1.510768E-02,  9.775367E-01,
+              2.391154E-01,  2.847920E-01,  5.592013E-01 ]
+
+    def test_component( self ) :
+
+        def verify_chunk( self, chunk ) :
+
+            # verify content
+            self.assertEqual( False, chunk.empty )
+            self.assertEqual( 35, chunk.length )
+            self.assertEqual( "DLW::KalbachMannDistributionData", chunk.name )
+
+            self.assertEqual( 2, chunk.NE )
+            self.assertEqual( 2, chunk.number_incident_energies )
+
+            self.assertEqual( 2, len( chunk.incident_energies ) )
+            self.assertAlmostEqual( 1.219437E+01, chunk.incident_energies[0] )
+            self.assertAlmostEqual( 20., chunk.incident_energies[1] )
+
+            self.assertAlmostEqual( 1.219437E+01, chunk.incident_energy(1) )
+            self.assertAlmostEqual( 20., chunk.incident_energy(2) )
+
+            self.assertEqual( 27, chunk.LOCC(1) );
+            self.assertEqual( 39, chunk.LOCC(2) );
+            self.assertEqual( 27, chunk.distribution_locator(1) );
+            self.assertEqual( 39, chunk.distribution_locator(2) );
```

Comment:
Yes. That is indeed the idea. Since locators are relative to the beginning of the superblock they are in instead of the beginning of the actual block they are in, I need to put in this locb nonsense. A user won't have to deal with it though since the user can just make the block with the default locb value (being 0). Internally, any block that has locb issues gets recreated whenever these internal locators have to change.

This is one of the things I would like to change once we go to a newer ACE format that keeps a flat array: all locators are relative to the beginning of the block they are defined in.